src/components/PreviewComment/index.js

```diff
@@ -109,12 +109,12 @@ export const styles = StyleSheet.create({
   container: (totalComment) => ({
     borderLeftWidth: 1,
     marginHorizontal: SIZES.base - 2,
-    borderLeftColor: totalComment >= 1 ? COLORS.balance_gray : COLORS.white
+    borderLeftColor: totalComment >= 1 ? COLORS.balance_gray : COLORS.almostBlack
   }),
   username: {
     fontFamily: fonts.inter[700],
     fontSize: normalizeFontSize(10),
-    color: COLORS.blackgrey,
+    color: COLORS.gray400,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color change from `COLORS.blackgrey` to `COLORS.gray400` for the username text might affect readability, especially on different backgrounds or in different light conditions. Please verify that this color provides sufficient contrast against its background.

```diff
-    color: COLORS.blackgrey,
+    color: COLORS.gray400,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->